### PR TITLE
Add option to log url request including query string

### PR DESCRIPTION
**Pull request by DrorBuhnik**
PR for #805

### CODE REVIEW

**Comment by jknack** [comment]
lovely! but we need a test :)

See [RequestLoggerTest](https://github.com/jooby-project/jooby/blob/72a7bd0b1cc2c7b92f8bd2717b529206f4c7b1f0/jooby/src/test/java/org/jooby/RequestLoggerTest.java)

File: jooby/src/test/java/org/jooby/RequestLoggerTest.java [hunk 1]
```diff
@@ -147,6 +169,13 @@ private Block path(final String path) {
     };
   }
 
+  private Block query(final String query) {
+    return unit -> {
+      Request req = unit.get(Request.class);
+      expect(req.queryString()).andReturn(Optional.of(query)).atLeastOnce();
```
Comment (DrorBuhnik) [inline jooby/src/test/java/org/jooby/RequestLoggerTest.java]: I'm not sure I like it that I'm the only one that uses .atLeastOnce() in the test
Any better way? (The RequestLogger call several times to queryString())

Comment (jknack) [inline jooby/src/test/java/org/jooby/RequestLoggerTest.java]: You could move the queryString() as part of the if block ... but I'm OK with this.

File: jooby/src/main/java/org/jooby/RequestLogger.java [hunk 2]
```diff
@@ -250,9 +250,8 @@ public void handle(final Request req, final Response rsp) throws Throwable {
       sb.append(Q).append(req.method());
       sb.append(SP);
       sb.append(req.path());
-      if (queryString && req.queryString().isPresent()) {
-        sb.append(QUERY);
-        sb.append(req.queryString().get());
+      if (queryString) {
+        req.queryString().ifPresent(s -> sb.append(QUERY).append(s));
```
Comment (DrorBuhnik) [inline jooby/src/main/java/org/jooby/RequestLogger.java]: I refactored into single queryString() call as you suggested
Better?

**Pull request merged**